src/config/zkevm.json

```diff
@@ -0,0 +1,12 @@
+{
+  "networkId": 1101,
+  "rpc": "https://polygonzkevm-mainnet.g.alchemy.com/v2/{{ALCHEMY_KEY}}",
+  "subgraph": "https://api.studio.thegraph.com/query/24660/balancer-polygon-zkevm-v2/v0.0.2",
+  "addresses": {
+    "nativeAsset": "",
+    "wrappedNativeAsset": "",
+    "vault": "",
+    "batchRelayerV3": "",
+    "batchRelayerV4": ""
+  }
```

Comment:
Sure, what about these details?

Reply:
They are only used for SOR /order endpoint. I'll add them in a separate PR for many chains after this with some other refactors (like making it just `batchRelayer`)